remote_run_script.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env/bash
+set -o errexit
+set -o nounset
+set -o xtrace
+
+# we assume that the script is ran from within the joblib_benchmarks
+# repository, so that there exists a launch script doing:
+# "git clone joblib_bencmarks && cd joblib_benchmarks"
+# "source remote_run_script.sh"
+
+# all repositories are created in the parent directory of joblib_benchmarks
+cd "${HOME}"
```

Comment:
This will mess up with the `HOME` directory of someone launching the script and it might conflit with existing config.
I would either use a temporary directory in `/tmp` or a temporary directory created in `joblib_benchmark` with `mkdir`.

Also, name the variable `BENCHMARK_TMP_DIR`.